examples/provider/provider-in-cluster.yaml

```
@@ -0,0 +1,33 @@
+---
+apiVersion: pkg.crossplane.io/v1
+kind: Provider
+metadata:
+  name: provider-kubernetes
+spec:
+  package: xpkg.upbound.io/crossplane-contrib/provider-kubernetes:v0.10.0
```

We should definitely add renovate to this repo, in the meantime let's start from the latest version:
```suggestion
  package: xpkg.upbound.io/crossplane-contrib/provider-kubernetes:v0.11.4
```